src/main/resources/templates/error/error.peb

```diff
@@ -0,0 +1,27 @@
+{# @pebvariable name="global" type="org.koil.view.GlobalViewModel" #}
+
+    {% extends 'layouts/dashboard-layout' %}
```

Comment:
Can we use a totally empty layout here? Literally the page with a back button? I'm worried that rendering the dashboard doesn't make sense in a lot of cases (e.g. a public form failing shouldn't show the logged in dashboard).

Reply:
Created a base-layout and navbar.